Makefile

```diff
@@ -256,6 +256,7 @@ test-scalability: gotestsum run-scalability
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) ./test/scalability/checker  \
 		--summary=$(SCALABILITY_RUN_DIR)/summary.yaml \
 		--cmdStats=$(SCALABILITY_RUN_DIR)/minimalkueue.stats.yaml \
+		--timeout=8m \
```

Comment:
Maybe 10m to be on the safer side? I guess if the test is interrupted we may not get the stats to investigate post-factum?

Reply:
The default is 5 , we should not double just because it did not fit in 5 min once.

Reply:
As you prefer, just pointing out I would be fine with a higher value, so that we don't lose data in some cases.

Reply:
Any reason to revert it? Are we expecting some flakes here still that we should address separately?